### PR TITLE
Antlr4 4.7.1-1 => 4.12.0

### PR DESCRIPTION
**Pull request by uberhacker**
Works properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/antlr4.rb [hunk 1]
```diff
@@ -3,39 +3,35 @@
...
-    system "install -Dm644 antlr-4.7.1-complete.jar #{CREW_DEST_LIB_PREFIX}/antlr-4.7.1-complete.jar"
-
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+  def self.build
     @antlrenv = <<~ANTLR_EOF
       # ANTLR (ANother Tool for Language Recognition) configuration
-      export CLASSPATH=\".:#{CREW_LIB_PREFIX}/antlr-4.7.1-complete.jar:\$CLASSPATH\"
-      alias antlr4=\"java -jar #{CREW_LIB_PREFIX}/antlr-4.7.1-complete.jar\"
-      alias grun=\"java org.antlr.v4.gui.TestRig\"
+      export CLASSPATH=".:#{CREW_PREFIX}/share/antlr/antlr-#{version}-complete.jar:$CLASSPATH"
```
Comment (satmandu) [inline packages/antlr4.rb:19]: We don't need `export` for variable declarations in env.d, since those operate under `set -a`.

Comment (uberhacker) [reply packages/antlr4.rb:19]: It doesn't hurt to include it for clarity but I can change if you insist.

Comment (uberhacker) [reply packages/antlr4.rb]: I just tested without the export and it still works as expected.

Comment (satmandu) [reply packages/antlr4.rb]: Fine either way.

**Review by satmandu — APPROVED** [review]

**Pull request merged**